cmd/zqd/listen/command.go

```diff
@@ -211,6 +214,9 @@ func (c *Command) initZeek() error {
 func (c *Command) initLogger() error {
 	if c.loggerConf == nil {
 		c.loggerConf = defaultLogger
+		for i := range c.loggerConf.Children {
+			c.loggerConf.Children[i].Level = c.logLevel
+		}
```

Comment:
Instead of adding this loop, it might be clearer to change defaultLogger to a method.
```go
func (c *Command) defaultLoggerConfig() &logger.Config {
	return &logger.Config{
		Type: logger.TypeWaterfall,
		Children: []logger.Config{
			{
				Name:  "http.access",
				Path:  "/dev/null",
				Level: c.logLevel,
			},
			{
				Path:  "stderr",
				Level: c.logLevel,
			},
		},
	}
}
```

Reply:
yeah, let's do this